src/test/java/com/configcat/EntrySerializationTest.java

```diff
@@ -19,25 +19,36 @@ void serialize() {
 
         String serializedString = entry.serialize();
 
-        assertEquals(String.format(SERIALIZED_DATA, fetchTime / 1000, "fakeTag", json), serializedString);
+        assertEquals(String.format(SERIALIZED_DATA, fetchTime, "fakeTag", json), serializedString);
     }
 
     @Test
-    void deserialize() throws Exception {
+    void payloadSerializationPlatformIndependent()  {
+        String payloadTestConfigJson  = "{\"p\":{\"u\":\"https://cdn-global.configcat.com\",\"r\":0},\"f\":{\"testKey\":{\"v\":\"testValue\",\"t\":1,\"p\":[],\"r\":[]}}}";
+
+        Config config = Utils.gson.fromJson(payloadTestConfigJson, Config.class);
+        Entry entry = new Entry(config, "test-etag", payloadTestConfigJson, 1686756435844L);
+        String serializedString = entry.serialize();
+
+        assertEquals(String.format(SERIALIZED_DATA, 1686756435844L, "test-etag", payloadTestConfigJson), serializedString);
```

Comment:
```suggestion
        assertEquals("1686756435844\ntest-etag\n" + payloadTestConfigJson, serializedString);
```

Please don't use `String.format` here to build the expected string because we want to make this test completely free of any platform-dependent behavior (e.g. from how Java formats a number to string).